docs/src/less/base/index.less

```diff
@@ -474,6 +474,24 @@ span.icon {
     margin: 1rem 0;
 }
 
+/* CHECKBOX & RADIO SECTION */
+
+// hack to override media query and force large size in example
+.demo-large-checkbox {
+    .checkbox__icon svg {
```

Comment:
Nit ~

We can just stick to standard CSS
```
.demo-large-checkbox .checkbox__icon svg {
}
```

[Same for lines 488 and 489]

Reply:
Ha! Yes, you got me ;-) Standard CSS is the best :-)

Reply:
:)